bin/bdii-update

```diff
@@ -271,7 +271,7 @@ def read_ldif(source):
         # Open pipe to LDIF
         if source[:7] == 'ldap://':
             url = source.split('/')
-            command = "ldapsearch -LLL -x -h %s -b %s 2>/dev/null" % (
+            command = "ldapsearch -LLL -x -H %s -b %s 2>/dev/null" % (
```

Comment:
I think this one is wrong, if you are splitting with '/', you will have only the host name here:

```
>>> source = "ldap://example.com:1234"
>>> source[:7] == "ldap://"
True
>>> url = source.split("/")
>>> url
['ldap:', '', 'example.com:1234']
>>>
```

Reply:
If you know for sure the URL has a path (which seems the case because of the `-b`)
```
>>> source = "ldap://example.com:1234/sad"
>>> url = source.rsplit("/", 1)
>>> url
['ldap://example.com:1234', 'sad']
>>>
```

Reply:
Thanks, I've just pushed a change adding the scheme (not very nice, but currently trying to limit changing the code)

Reply:
seems reasonable. Probably this should be done with urllib.parse or similar but that would imply some larger refactoring